docs/gallery/general/nwb_gallery_config.yaml

```
@@ -0,0 +1,10 @@
+namespaces:
+  core:
+    version: 2.7.0-alpha
```

Can you remind me, when NWB schema 2.8.0 is released and used by PyNWB, will this work, or do we (and users) need to constantly keep this up to date? Will the user get an error if there is a version mismatch?

---

As of right now, we don't do anything with the version. It is purely a best practice until we decide to enforce a check

---

Got it. Thanks. I think we should never point to unreleased versions of the schema. Can we make this change?
```suggestion
    version: 2.7.0
```